pkg/controller/jaeger/deployment.go

```diff
@@ -0,0 +1,95 @@
+package jaeger
+
+import (
+	"context"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/jaegertracing/jaeger-operator/pkg/apis/io/v1alpha1"
+	"github.com/jaegertracing/jaeger-operator/pkg/inventory"
+)
+
+func (r *ReconcileJaeger) applyDeployments(jaeger v1alpha1.Jaeger, desired []appsv1.Deployment) error {
+	opts := client.MatchingLabels(map[string]string{
+		"app.kubernetes.io/instance":   jaeger.Name,
+		"app.kubernetes.io/managed-by": "jaeger-operator",
+	})
+	depList := &appsv1.DeploymentList{}
+	if err := r.client.List(context.Background(), opts, depList); err != nil {
+		return err
+	}
+
+	logFields := log.WithFields(log.Fields{
+		"namespace": jaeger.Namespace,
+		"instance":  jaeger.Name,
+	})
+
+	// we now traverse the list, so that we end up with three lists:
+	// 1) deployments that are on both `desired` and `existing` (update)
+	// 2) deployments that are only on `desired` (create)
+	// 3) deployments that are only on `existing` (delete)
+	depInventory := inventory.ForDeployments(depList.Items, desired)
+	for _, d := range depInventory.Create {
+		logFields.WithField("deployment", d.Name).Debug("creating deployment")
```

Comment:
you pass the "deployment" twice into the log message. Maybe better to add it to fields.  I think this is in all files in this package.

Reply:
You mean the word "deployment"?

Reply:
yes